src/configuration/structures/structures.ts

```diff
@@ -60,6 +67,14 @@ export interface Bot {
   work_scheduler?: WorkScheduler;
 }
 
+export interface BotTemplate {
+  id: string;
+  name?: string;
+  avatar?: string;
+  default_group_priority?: GroupPriority;
+  job_title?: string;
```

Comment:
```suggestion
  id: string;
  name?: string;
  avatar?: string;
  max_chats_count?: number;
  default_group_priority?: GroupPriority;
  job_title?: string;
```
Should we add `max_chats_count` property here like in lc-sdk-go?

https://github.com/livechat/lc-sdk-go/blob/b81f8a540508e0a86cb3834e23bafcfd5f76a14f/configuration/structures.go#L121